Steffy/Assignment3.1.py

```diff
@@ -0,0 +1,9 @@
+firstname=input("Enter a First Name: ")
+print(firstname)
```

Comment:
you can do this by assigning a variable here
e.g
length_first_name = len(firstname)